sass/settings/_variables.scss

```diff
@@ -1,3 +1,6 @@
 $font-family-base: 'Roboto', Arial, sans-serif;
 
-$navbar-light-toggler-border-color: none;
\ No newline at end of file
+$navbar-light-toggler-border-color: none;
+
+$footer-font-size: 1.125rem;
+$footer-height: 5.625rem;
```

Comment:
Separating component properties in variables is useful. Let's discuss in which use-cases is good to do it and in which - it isn't.